sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -2265,6 +2268,20 @@ private String getNumberOfDosesFromVaccinations(Vaccination vaccination) {
 		return vaccination != null ? vaccination.getVaccineDose() : "";
 	}
 
+	private Date simplyDate(Date date) {
```

Comment:
```suggestion
	private Date removeTime(Date date) {
```

Also, please move this method to the `DateHelper` class